utils/mock-data-loader/temporary-token-handler.js

```diff
@@ -13,11 +13,11 @@ module.exports = async (user) => {
     },
     data: user,
   }).catch((err) => {
-    console.log('failed to create temp user');
-    console.log(`${JSON.stringify(err)}`);
+    console.info('failed to create temp user');
+    console.info(`${JSON.stringify(err)}`);
```

Comment:
should these 2 get merged into 1 console? we may need to use `console.error` here?

Reply:
Good shout, will do.

Reply:
Updated.